NEWS

```diff
@@ -1,3 +1,14 @@
+25-Nov-2020 Ignition 2.8.0
+
+  Features:
+
+    - Support unmasking systemd units
+
+  Changes:
+
+    - Switch system base config from single file to .d directory
+    - Add Go merge API producing a transcript of merge operations
```

Comment:
I've gone back and forth on this, but `that produces` might be clearer than `producing`.